packages/backend/test/attesterDescription.test.mjs

```diff
@@ -0,0 +1,113 @@
+import { wallet } from '../src/config.mjs'
+import './server.mjs'
+import fetch from 'node-fetch'
+import { ethers } from 'ethers'
+import { expect } from 'chai'
+import randomf from 'randomf'
+import { clearCollection } from '../src/index.mjs'
+
+const random = () => Math.floor(Math.random() * 100000)
+const randomSignature = () =>
+  '0x00' + randomf(2n << (BigInt(512) - 1n)).toString(16)
+const load = 1000 // time before server
+
+describe('Attester Description Tests', function () {
+  let attesters
+  let headers
+  this.timeout(3000)
+  before(async () => {
+    await new Promise((r) => setTimeout(r, load))
+
+    const url = new URL('/api/unirep/attesters', process.env.HTTP_SERVER)
+    const r = await fetch(url.toString()).then((r) => r.json())
+    attesters = r.items.map((x) => '0x' + parseInt(x._id).toString(16))
+  })
+
+  beforeEach(async () => {
+    await new Promise((r) => setTimeout(r, load))
+    await clearCollection('AttesterDescription', {
+      where: { _id: attesters[-1] },
+    })
+    headers = {
+      description: 'example description',
+      icon: '<svg>...</svg>',
+      url: 'https://developer.unirep.io',
+      name: 'unirep',
+      network: 'unirep-goerli',
+      nonce: random(),
+    }
+  })
+
+  it('should not update info with incorrect signature', async () => {
+    headers.signature = randomSignature()
```

Comment:
I think here could be a random wallet to sign the message
```ts
    const randomWallet = new ethers.Wallet.createRandom()
    const hash = ethers.utils.solidityKeccak256(
      ['uint256', 'string'],
      [headers.nonce, headers.description]
    )

    headers.signature = await randomWallet.signMessage(ethers.utils.arrayify(hash))
```

Reply:
fixed